src/Shared/PlatformSupport/AssetLoader.cs

```diff
@@ -239,7 +220,6 @@ private AssemblyDescriptor GetAssembly(string name)
                     throw new InvalidOperationException(
                         $"Assembly {name} needs to be referenced and explicitly loaded before loading resources");
 #else
-                    name = Uri.UnescapeDataString(name);
```

Comment:
Introduced because of this https://github.com/AvaloniaUI/Avalonia/issues/2544, but worth double-checking before actually deleting